plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/impls/materialize/MaterializeSchema.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ext.postgresql.model.impls.materialize;
+
+import java.sql.SQLException;
+
+import org.jkiss.dbeaver.ext.postgresql.PostgreConstants;
+import org.jkiss.dbeaver.ext.postgresql.model.PostgreDatabase;
+import org.jkiss.dbeaver.ext.postgresql.model.PostgreRole;
+import org.jkiss.dbeaver.ext.postgresql.model.PostgreSchema;
+import org.jkiss.dbeaver.model.exec.jdbc.JDBCResultSet;
+
+public class MaterializeSchema extends PostgreSchema {
+    public MaterializeSchema(PostgreDatabase owner, String name, JDBCResultSet resultSet) throws SQLException {
+        super(owner, name, resultSet);
+    }
+
+    public MaterializeSchema(PostgreDatabase database, String name, PostgreRole owner) {
+        super(database, name, owner);
+    }
+
+    @Override
+    public boolean isSystem() {
+        return isCatalogSchema() || PostgreConstants.INFO_SCHEMA_NAME.equalsIgnoreCase(name)
+                || name.startsWith(PostgreConstants.SYSTEM_SCHEMA_PREFIX) || name.startsWith("mz_");
```

Comment:
We use a 4-space continuation indent. Additionally, it's worth breaking expression into multiple lines to simplify reading:
```suggestion
        return isCatalogSchema()
            || PostgreConstants.INFO_SCHEMA_NAME.equalsIgnoreCase(name)
            || name.startsWith(PostgreConstants.SYSTEM_SCHEMA_PREFIX)
            || name.startsWith("mz_");
```